signature/policy_eval_signedby_test.go

```diff
@@ -113,14 +117,17 @@ func TestPRSignedByIsSignatureAuthorAccepted(t *testing.T) {
 	pr, err = NewPRSignedByKeyData(ktGPG, []byte{}, prm)
 	require.NoError(t, err)
 	// Pass nil pointers to, kind of, test that the return value does not depend on the parameters.
-	sar, parsedSig, err = pr.isSignatureAuthorAccepted(nil, nil)
-	assertSARRejectedPolicyRequirement(t, sar, parsedSig, err)
+	sar, parsedSig, err = pr.isSignatureAuthorAccepted(m, nil, nil)
+	// FIXME: The error is: errors.InvalidArgumentError("no armored data found") but for
+	// some reason it is surfaced here and not translated to PolicyRequirementError.
+	// However the sar is rejected...
+	assertSARRejectedPolicyRequirement(t, sar, parsedSig, PolicyRequirementError(""))
```

Comment:
Well, gpgme happens to accept empty bytes (the `[]byte{}` in `NewPRSignedByKeyData`) as valid input, “correctly importing” it as a set of zero keys.  (It also happens to accept completely invalid input, but that’s beside the point).

With gpgme, the code path fails in the `	if len(trustedIdentities) == 0 {` case, which is a `PolicyRequirementError`; with openpgp, it fails in the `… err := mechanism.ImportKeysFromBytes(); if err != nil` case, which is an operation error (like `ioutil.ReadFile` failing). Though arguably the `len(trustedIdentities) == 0` is an invalid `PolicyRequirement`, not a failure to fulfill it… it really isn’t that well defined.

I guess, considering https://tools.ietf.org/html/rfc4880#section-3.6 , it could be argued that an empty keyring is invalid… We will need to nail these things down.